sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Rules/InheritedCollidingInterfaceMembers.cs

```diff
@@ -131,14 +131,15 @@ private static bool IsNotEventRemoveAccessor(IMethodSymbol methodSymbol2)
             var names = collidingMembers
```

Comment:
I think it makes sense to address also the sonar source code smell and use IEnumerable: https://sonarcloud.io/project/issues?id=sonaranalyzer-dotnet&issues=AXDyLJsTbpzdS9P4et1k&open=AXDyLJsTbpzdS9P4et1k&pullRequest=3236